### PR TITLE
Port to MiniScaffold

### LINKED ISSUE #10: Support .NET Standard
What do you think about targeting this library to `netstandard1.6` and moving project file to new .NET SDK?

### PR DESCRIPTION
**Pull request by gsomix**
Closes #10 

- [x] Drop F#3.1 support
- [x] Port to MiniScaffold (new SDK projects, net461 and netstandard targets)
- [x] Fix tests
- [x] Add targets for building documentation

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+language: csharp
+sudo: required
+dist: trusty
 
-env:
-  matrix:
-    - MONO_VERSION="3.12.0"
+dotnet: 2.0.3
+mono:
+  - 5.4.1
```
Comment (dsyme) [inline .travis.yml:7]: I'm not sure we need so many Mono variations for this little library

**Comment by dsyme** [comment]
@gsomix This looks great. Please do complete the next two steps and then we can pull it

**Comment by gsomix** [comment]
@dsyme Ok, could you please check if I forgot something?

**Comment by gsomix** [comment]
I have rebased branch so it's ready to merge again.

**Comment by sergey-tihon** [comment]
@dsyme May I merge this PR?

**Comment by dsyme** [comment]
Yes, go for it

**Comment by sergey-tihon** [comment]
@gsomix I see that you've changed the project name, so this also means the new name for NuGet package. Is it expected within the scope of current PR? (I guess no, probably we want to release it as the new version of `FSharp.Core.Fluent-4.0`)

**Comment by sergey-tihon** [comment]
Thank you @gsomix !!!

@dsyme could you please push the new package to NuGet? (i do not have rights)

**Comment by gsomix** [comment]
@dsyme @sergey-tihon Thank you guys!

**Comment by dsyme** [comment]
@sergey-tihon Have added you as owner

I don't mind if package name changes to FSharp.Core.Fluent (dropping -4.0).  But I'll leave it to you guys to decide

@gsomix Would you like to become a co-owner on this package?

**Comment by gsomix** [comment]
@dsyme That would be great.

**Comment by dsyme** [comment]
What's your nuget handle?

**Comment by gsomix** [comment]
@dsyme 
https://www.nuget.org/profiles/gsomix

**Comment by sergey-tihon** [comment]
@gsomix you've been added as an owner

**Pull request merged**